daisy/test_common_test.go

```diff
@@ -215,6 +218,9 @@ func newTestGCEClient() (*daisyCompute.TestClient, error) {
 		return []*compute.ForwardingRule{{Name: testForwardingRule}}, nil
 	}
 	c.ListLicensesFn = func(p string, _ ...daisyCompute.ListCallOption) ([]*compute.License, error) {
+		if p == p403 {
+			return nil, &googleapi.Error{Code: 403}
```

Comment:
Why is this needed? 

Do we need to handle other error codes in the same way?

Reply:
This is a test for this scenario, were testing whether we handle the 403 correctly.

Reply:
The PR doesn't fix the issue but instead bypass the check. The root cause was https://github.com/GoogleCloudPlatform/compute-image-tools/issues/1187 incorrectly changed the required license permission from "get" to "list". Some GCP projects, e.g. `vm-options`, may allow anyone to "get" the specific license, but they don't allow anyone to "list" all available licenses hosted on their projects.

Reply:
Is this 'get' specific license acl new? This bypass was specifically created to solve the license issue as you can have no access to a license (not get access) but can still use it when creating an image. If there is a specific get acl now we can update the logic to run get if list fails, but that's a separate change. This change just fixes the IgnoreLicenseValidationIfForbidden option.

Reply:
I just tested and this project has no special GET permsions as far as I can tell, I get the same error:
APIError: googleapi: Error 403: Required 'compute.licenses.get' permission for 'projects/vm-options/global/licenses/enable-vmx', forbidden

Reply:
Also moving to the list call vs the get call was because licenses.list was not available when first implemented but it is now. All the cache checks use the list call so this was bringing the license lookup inline. If fallback to get calls is wanted we can implement that across the board.